run/hello-broken/test/system.test.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC. All rights reserved.
```

Comment:
Incomplete Apache license header.

Reply:
No offense, but the current license header or previous ones do not matter.
OSPO requires an unmodified version of the header; there is no shortened version:
go/releasing/preparing#license-headers

Many PRs and projects don't fully follow the rules, but if we are trying to add a header here, we should use the full form as described in the doc above.

Reply:
Fair, I am trying to adhere to whatever standard this repo prefers. Will revise files changed in my currently open PRs.